pkg/engine/renderer/dependencies_test.go

```diff
@@ -107,6 +107,9 @@ func TestSetDependenciesHash(t *testing.T) {
 			assert.NilError(t, err)
 			assert.DeepEqual(t, pod("somename", "namespace"), p)
 		}},
+		{name: "no change in unstructured CRD", obj: unstructuredCrd("crd", "namespace"), assert: func(us *unstructured.Unstructured) {
```

Comment:
So there is no easy way to collect the CRDs as part of the diagnostics (or cleanup for that matter) 😞

Reply:
You mean because it didn't change? That's not correct, this test is specifically about the dependencies hash - no dependencies for CRDs, therefore no change.

I've extended one test in `enhancer_test.go` to verify that and make it more obvious that the top-level annotations in `metadata/annotations` and `metadata/labels` are always added.

Reply:
Got it!